### ISSUE TITLE
Incorrect outdented heredoc line warning

### ISSUE THREAD

**Issue by AntoineAugusti** [rrrene/credo#924]
### Environment

* Credo version (`mix credo -v`): `1.5.6-ref.master.55344099+uncommittedchanges`
* Erlang/Elixir version (`elixir -v`): `Elixir 1.12.2`
* Operating system: macOS

### What were you trying to do?

Running
```
mix credo --strict 
```

### Expected outcome

No warnings

### Actual outcome

Warning that does not seem appropriate
```
warning: outdented heredoc line. The contents inside the heredoc should be indented at the same level as the closing """. The following is forbidden:

    def text do
      """
    contents
      """
    end

Instead make sure the contents are indented as much as the heredoc closing:

    def text do
      """
      contents
      """
    end

The current heredoc line is indented too little
  apps/transport/lib/transport/file_downloads.ex:3
```

Here is the relevant file and lines https://github.com/etalab/transport-site/blob/1b71f63c3cfe4b61958c1d510a92b0a4414c91b3/apps/transport/lib/transport/file_downloads.ex#L1-L5

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by AntoineAugusti** [comment]
Can confirm this is fixed! Thanks a lot.

**Comment by rrrene** [comment]
This is now live in `v1.6.3`. Thx again!
